packages/SwingSet/src/initializeSwingset.js

```diff
@@ -278,7 +278,10 @@ export async function initializeSwingset(
 
   // Use ambient process.env only if caller did not specify.
   const { env: { SWINGSET_WORKER_TYPE } = process.env } = runtimeOptions;
-  const defaultManagerType = config.defaultManagerType || SWINGSET_WORKER_TYPE;
+
+  // Override the worker type if specified by the caller, to avoid having to
+  // edit the config just to run everything under a different manager.
+  const defaultManagerType = SWINGSET_WORKER_TYPE || config.defaultManagerType;
```

Comment:
With this priority flip, could/should we remove the similar line from packages/solo/src/init-basedir.js ?

Reply:
I think we still want the caller of `ag-solo init` to be able to override the default `defaultManagerType`.